autofill-parser/src/main/java/com/github/androidpasswordstore/autofillparser/AutofillFormParser.kt

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright © 2014-2020 The Android Password Store Authors. All Rights Reserved.
+ * SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
I have spent some more time on license research and would propose changing this (and all other occurrences) to the following:
```suggestion
 * SPDX-License-Identifier: GPL-3.0-only OR (LGPL-3.0-only WITH LGPL-3.0-linking-exception)
```

This is because

1. We are still bound by the GPL on the existing code whether or not we put GPL in here or not, so making this explicit via an `OR` directive only adds transparency.
2. Since LGPL was created mainly with binary shared libraries written in C in mind, it does not cleanly apply to Java and especially Android libraries, which (as one can argue) are always linked statically. Some say that it's fine as long as you provide decompiling steps using dex2jar, but it is obviously not our intention to impose such unnecessary restrictions on projects that want to use the library. IANAL, but this potentially contentious point appears to be clarified sufficiently by the Linking Exception to the LGPL. 

@msfjarvis @Valodim What do you think?

Reply:
I think your reasoning is pretty sound and the change looks good to me

Reply:
Part 1 is not true. The original copyright holders are not bound to the GPL. This is also true in the US.

Dual licensing GPL+LGPL doesn't make a lot of sense, since LGPL grants the user more rights in comparison to GPL, so we as the users choose to use the license under LGPL and will ignore the GPL. Check https://opensource.stackexchange.com/q/8401

You can still choose to do so, technically there is no difference for us users of your library.

Reply:
> Part 1 is not true. The original copyright holders are not bound to the GPL. This is also true in the US.
> 
> Dual licensing GPL+LGPL doesn't make a lot of sense, since LGPL grants the user more rights in comparison to GPL, so we as the users choose to use the license under LGPL and will ignore the GPL. Check https://opensource.stackexchange.com/q/8401
> 
> You can still choose to do so, technically there is no difference for us users of your library.

I guess you are right, I wasn't aware of what the third paragraph in this answer (https://opensource.stackexchange.com/a/8404) says: LGPL doesn't just give you strictly more leeway in how you use the library, it explicitly allows you to use it under GPL as well.

I will remove the superfluous GPL modifiers then.